src/rebar_app_utils.erl

```diff
@@ -291,7 +291,7 @@ update_source(AppInfo, {pkg, PkgName, PkgVsn, Hash}, State) ->
                                         ?PACKAGE_TABLE, State) of
         {ok, Package, RepoConfig} ->
             #package{key={_, PkgVsn1, _},
-                     checksum=Hash1,
+                     outer_checksum=Hash1,
```

Comment:
It's arguable whether to change this to outer_checksum or keep it as checksum and abstracted from hex.